src/System.Management.Automation/engine/PropertyAccessor.cs

```diff
@@ -166,11 +166,11 @@ internal JsonConfigFileAccessor()
         /// not interfere with PowerShell initialization
         /// </summary>
         /// <returns>Returns the directory if present or creatable. Throws otherwise.</returns>
-        private string GetAppDataConfigDirectory()
+        private string GetCurrentUserConfigDirectory()
         {
             if (null == appDataConfigDirectory)
```

Comment:
See line 159. initialization is conditional on the directory existing or creatable. The check enables PowerShell to initialize even if it cannot create the directory.